src/Api/StructureShape.php

```diff
@@ -24,12 +29,11 @@ public function __construct(array $definition, ShapeMap $shapeMap)
      */
     public function getMembers()
     {
-        $result = [];
-        foreach ($this->definition['members'] as $name => $definition) {
-            $result[$name] = $this->shapeFor($definition);
+        if (empty($this->members)) {
```

Comment:
I think Graham is saying that you can omit the `empty` function and just call `if ($this->members) {`